services/cd-service/pkg/repository/transformer.go

```diff
@@ -129,6 +135,7 @@ func (t TransformerFunc) Transform(ctx context.Context, fs billy.Filesystem) (st
 var _ Transformer = TransformerFunc(func(_ context.Context, _ billy.Filesystem) (string, error) { return "", nil })
 
 type CreateApplicationVersion struct {
+	Version        uint64
```

Comment:
How does the `publish.sh` script know what version to put here?

Reply:
> do we need to make sure that there was no existing code (outside of kuberpult) calling /release api and expecting 204 (and just just 20x)

Yes. I actually checked that. The existing scripts take any 200 as success.